pkg/matcher/cel.go

```diff
@@ -46,3 +61,49 @@ func celEvaluate(expr string, event *info.Event) (ref.Val, error) {
 	}
 	return out, nil
 }
+
+type celPac struct {
+	vcx   provider.Interface
+	ctx   context.Context
+	event *info.Event
+}
+
+func (t celPac) ProgramOptions() []cel.ProgramOption {
+	return []cel.ProgramOption{
+		cel.Functions(
+			&functions.Overload{
+				Operator: "pathChanged",
+				Unary:    t.pathChanged,
+			},
+		),
+	}
+}
+
+func (t celPac) pathChanged(vals ref.Val) ref.Val {
+	var match types.Bool
+	fileList, err := t.vcx.GetFiles(t.ctx, t.event)
+	if err != nil {
+		return types.Bool(false)
+	}
+	for i := range fileList {
+		if v, ok := vals.Value().(string); ok {
+			g := glob.MustCompile(v)
+			if g.Match(fileList[i]) {
+				return types.Bool(true)
+			}
+		}
+		match = types.Bool(false)
+	}
+
```

Comment:
@chmouel I think these changes are not required here 

I believe you added changes on existing code copy

Reply:
yeah this PR is a followup to #751